### PR TITLE
OS-8370 Welcome MNX (fix loader & onbld copyright)

### PR DESCRIPTION
**Pull request by danmcd**

### CODE REVIEW

**Comment by danmcd** [comment]
This needs more fixes, as the triton-logo.png is not displaying yet.

File: usr/src/boot/forth/brand-smartos.4th [hunk 1]
```diff
@@ -21,17 +22,17 @@
...
-	s" #     # #    #   #   #      #  # #   #    " brand+
-	s" #     # #    #   #   #      #   ##   #    " brand+
-	s"  #####   ####    #   ###### #    #   #  TM" brand+
+	s"                                           " brand+
+	s" ####### #####  # #######   ###   #    #   " brand+
+	s"    #    #      #    #     #   #  ##   #   " brand+
+	s"    #    #      #    #    #     # # #  #   " brand+
+	s"    #    #      #    #    #     # #  # #   " brand+
+	s"    #    #      #    #     #   #  #   ##   " brand+
+	s"    #    #      #    #      ###   #    # TM" brand+
```
Comment (bahamat) [inline usr/src/boot/forth/brand-smartos.4th:35]: I stylized the text logo a bit in a way that I like better:

For posterity, this is a slight modification to the clr6x8 figlet font.

```
        s"                                           " brand+
        s" #####  ####   #  #####   ###   #   #      " brand+
        s"   #    #   #  #    #    #   #  ##  #      " brand+
        s"   #    ####   #    #    #   #  # # #      " brand+
        s"   #    # #    #    #    #   #  #  ##      " brand+
        s"   #    #  #   #    #    #   #  #  ##      " brand+
        s"   #    #   #  #    #     ###   #   # TM   " brand+
```

File: usr/src/tools/onbld/Checks/Copyright.py [hunk 2]
```diff
@@ -38,7 +39,7 @@ def is_copyright(line):
 	return re.search(r'Copyright (?!\[yyyy\])', line)
 
 def is_current_copyright(line):
-	return re.search(r'Copyright %s Joyent, Inc.$' % time.strftime('%Y'), line)
+	return re.search(r'Copyright %s MNX Cloud, Inc.$' % time.strftime('%Y'), line)
```
Comment (bahamat) [inline usr/src/tools/onbld/Checks/Copyright.py:42]: Line 61 also needs updating

**Review by bahamat — CHANGES_REQUESTED** [review]

File: usr/src/boot/forth/brand-smartos.4th (hunk 1 above)
Comment (danmcd) [reply usr/src/boot/forth/brand-smartos.4th]: Will fix.

File: usr/src/tools/onbld/Checks/Copyright.py (hunk 2 above)
Comment (danmcd) [reply usr/src/tools/onbld/Checks/Copyright.py:42]: Will fix.

**Review by bahamat — APPROVED** [review]

**Comment by danmcd** [comment]
Tested by making an ISO and seeing the installer.  Also did "git pbchk" with a new-world-order /opt/onbld on this very commit! Only expected whitespace nits were reported.

**Pull request merged**